release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Added 'update incident' action
```

Comment:
I think we should be adding the PAPP ticket to the end of this line

Reply:
Done